sympy/core/symbol.py

```diff
@@ -205,10 +205,25 @@ def numbered_string_incr(s, start=0):
 
 class Symbol(AtomicExpr, Boolean):
     """
+    A Symbol is an expression of a mathematical variable using a
+    name.
+
+    Symbols support subscripts and Greek and Latin characters.
+    Symbols can be combined to form SymPy expressions. A single Symbol
+    can be created using the Symbol() constructor but to create
+    multiple symbols in one statement use the symbols() function.
+
+
     Assumptions:
        commutative = True
 
+    There are more assumptions for a Symbol whose default value is set to
+    ``None``. To read more about them see :ref: `assumptions
+    <https://docs.sympy.org/latest/guides/assumptions.html#predicates>`_.
```

Comment:
This should be a sphinx reference rather than a URL. A sphinx reference will not include the `sympy.org` domain name so that the docs can be built locally and still work.